web_note_test/__init__.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
```

Comment:
don't add a demo module on this level. Put it into your actual module, if somebody want to try this out, this person can symlink it into the addons path